pyscf/md/test/test_velocverlet.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# Copyright 2014-2018 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+from pyscf import gto, scf
+import pyscf.md as md
+
+CHECK_STABILITY = False
+
+h2o = gto.M(verbose=3,
+            output='/dev/null',
+            atom=[['O', 0, 0, 0], ['H', 0, -0.757, 0.587],
+                  ['H', 0, 0.757, 0.587]],
+            basis='def2-svp')
+
+hf_scanner = scf.RHF(h2o)
+hf_scanner.build()
+hf_scanner.conv_tol_grad = 1e-6
+hf_scanner.max_cycle = 700
+
+ethylene = gto.M(verbose=3,
+                 output='/dev/null',
+                 atom=''' C -0.0110224 -0.01183 -0.0271398
+    C -0.000902273 0.0348566 1.34708
+    H 1.07646 0.0030022 -0.456854
+    H 0.976273 -0.140089 1.93039
+    H -0.926855 -0.147441 1.98255
+    H -0.983897 0.0103535 -0.538589
+    ''',
+                 unit='ANG',
+                 basis='ccpvdz',
+                 spin=0)
+
+hf = ethylene.RHF().run()
+ncas, nelecas = (2, 2)
+casscf_scanner = hf.CASSCF(ncas, nelecas)
+casscf_scanner.conv_tol = 1e-12
+casscf_scanner.conv_tol_grad = 1e-6
+
+
+def tearDownModule():
+    global h2o, hf_scanner, ethylene, hf, casscf_scanner
+    h2o.stdout.close()
+    ethylene.stdout.close()
+    hf_scanner.stdout.close()
```

Comment:
It may close the same file twice. You probably don't need to close `hf_scanner.stdout` and `casscf_scanner.stddout`.